ZenPacks/zenoss/LinuxMonitor/modeler/plugins/zenoss/cmd/linux/lvm.py

```diff
@@ -99,10 +99,10 @@ def process(self, device, results, log):
         section = ''
         hd_re = re.compile('(?P<disk>\S+) (?P<size>\d+)')
         dev_blk_re = re.compile('(?P<device_block>.*)\s+(?P<major_minor>\d+:\d+)\s+\d+\s+(?P<size>\d+)\s+\d+\s+(?P<type>\w+)\s*(?P<mount>\S*)')
-        pv_re = re.compile('\s*(?P<pv_name>\S+)\s*(?P<pv_fmt>\S+)\s*(?P<pv_attr>\S+)\s*(?P<pv_size>\S+)'
-                           '\s*(?P<pv_free>\S+)\s*(?P<pv_uuid>\S+)\s*(?P<vg_name>\S*)')
-        vg_re = re.compile('\s*(?P<vg_name>\S+)\s*(?P<vg_attr>\S+)\s*(?P<vg_size>\S+)\s*(?P<vg_free>\S+)\s*(?P<vg_uuid>\S+)')
-        lv_re = re.compile('\s*(?P<lv_name>\S+)\s*(?P<vg_name>\S+)\s*(?P<lv_attr>\S+)\s*(?P<lv_size>\S+)\s*(?P<lv_uuid>\S+)\s*(?P<origin>\S*)')
+        pv_re = re.compile('(?P<pv_name>\S+)\s*(?P<pv_fmt>\S+)\s*(?P<pv_attr>\S+)\s*(?P<pv_size>\d+)'
```

Comment:
As long as we're refactoring these Regex's I'd recommend breaking them up for maximum readability.
This makes is much easier to spot problems. For example, for the dev_blk_re, you can write as:

<pre>
        dev_blk_re = re.compile('(?P<device_block>.*)'        # Block Devices   
                                  '\s+(?P<major_minor>\d+:\d+)' # Blk Major/Minor 
                                  '\s+\d+\s+'                   # RM number       
                                  '(?P<size>\d+)'               # Size            
                                  '\s+\d+\s+'                   # RO number       
                                  '(?P<type>\w+)'               # Block Type      
                                  '\s*'                         # Unlimited space 
                                  '(?P<mount>\S*)')             # Mount point
</pre>